bundle/internal/schema/annotations_openapi_overrides.yml

```diff
@@ -0,0 +1,155 @@
+github.com/databricks/cli/bundle/config/resources.Cluster:
```

Comment:
We can keep them so that they are searchable. If we don't include it it looks like some subset of descriptions are missing (which they are). I think having them show up as placeholders is better because it is explicit.